src/eye_extractor/dr/venous_beading.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from eye_extractor.common.negation import is_negated
+from eye_extractor.common.severity import extract_severity, Severity
+from eye_extractor.laterality import build_laterality_table, create_new_variable
+
+VEN_BEADING_PAT = re.compile(
+    r'\b('
+    r'venous beading|vb'
+    r')\b',
+    re.I
+)
+
+
+def get_ven_beading(text: str, *, headers=None, lateralities=None) -> list:
+    if not lateralities:
+        lateralities = build_laterality_table(text)
+    data = []
+    for new_var in _get_ven_beading(text, lateralities, 'ALL'):
+        data.append(new_var)
+    if headers:
+        pass
+
+    return data
+
+
+def _get_ven_beading(text: str, lateralities, source: str) -> dict:
+    for m in VEN_BEADING_PAT.finditer(text):
+        negwords = {'no', 'or', 'neg', 'without', 'w/out', '(-)'}

```

Comment:
We may want to consider merging all of these scattered `negwords` lists -- I'd assume, in most cases, these could be shared. Maybe, if not otherwise specified, a default set of `negwords` is used.

Reply:
Great point, it will help with our generalizability. Similar to the hemorrhage refactor, I'm excited to start drying out the code base.